src/components/SortPanel.jsx

```diff
@@ -0,0 +1,41 @@
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { Link, useSearchParams } from 'react-router-dom'
+
+export default function SortPanel ({ order, toggleOrder }) {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const topic = searchParams.get("topic");
+
+  const path = `/articles/${topic ? `topic=${topic}` : ''}`
```

Comment:
I've deleted this now. It was from an earlier attempt to deal with the search params